doc/api/fs.md

```diff
@@ -2318,7 +2318,8 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `flags` {string|number} See [support of file system `flags`][].
+* `flags` {string|number} **Default:** `'r'`
```

Comment:
Nits:
1. Missing period.
2. One unneeded space in the next line indent alignment,
3. It seems our common style is to place `**Default:**` notes at the end of a parameter description:

```md
* `flags` {string|number} See [support of file system `flags`][].
  **Default:** `'r'`.
```